miprometheus/problems/seq_to_seq/video_text_to_class/cog/cog.py

```diff
@@ -774,6 +774,8 @@ def collect_statistics(self, stat_col, data_dict, logits):
 		stat_col['acc'] = acc_total
 		stat_col['acc_answer'] = acc_answer
 		stat_col['acc_pointing'] = acc_pointing
+		if self.set == 'test' :
```

Comment:
is there any reason why we shouldn't collect those accuracies for training/validation sets as well?

Reply:
Hmm, no particular reason, it was just to avoid a massive 44 accuracies printing during training. But I can do it if you prefer?

Reply:
how else you want to get those statistics when training?